src/applications/disability-benefits/all-claims/pages/reservesNationalGuardService.jsx

```diff
@@ -181,11 +32,10 @@ export const schema = {
       properties: {
         reservesNationalGuardService: {
           type: 'object',
+          required: ['obligationTermOfServiceDateRange', 'unitName'],
```

Comment:
:thinking: Should this be in the schema defined for the front end only, or in the schema shared with the back end?

Reply:
🤔 you're right, probably the schema

Reply:
Actually, I'm not really sure. We're just cherry picking properties from the schema, so the `required` doesn't get used. Maybe they should be in both places?